src/main/scala/com/paulgoldbaum/influxdbclient/Database.scala

```diff
@@ -12,12 +13,12 @@ class Database protected[influxdbclient]
   with RetentionPolicyManagement
   with DatabaseManagement
 {
-  def write(point: Point,
+  def write[A](point: A,
```

Comment:
I have concern about keeping binary compatibility. @paulgoldbaum do you have any thoughts ?